src/index.js

```diff
@@ -1098,20 +1108,18 @@ class Enmap extends Map {
   }
 
   /**
-   * Internal method used to validate persistent enmap names (valid Windows filenames)
+   * INTERNAL METHOD used to validate persistent enmap names (valid Windows filenames)
```

Comment:
If we're being consistent 
```suggestion
   * INTERNAL METHOD. Validates persistent enmap names (valid Windows filenames)
```